tests/integration.py

```diff
@@ -383,6 +384,44 @@ def test_multiple():
                 return Test.PASS
 
 
+
+    def test_multithreaded():
+        """ tests if multithreading works correctly
+        """
+        config = {
+                'filename': 'temp_config',
+                'port': 8080,
+                'handlers': [
+                    ('/delay','DelayHandler',[]),
```

Comment:
Maybe assign a value to the delay, and then use that in the check instead of hardcoding `time2-time1 > 2.0`